src/hello_app.py

```diff
@@ -1,13 +1,21 @@
 #this script set up the basic flask app
-
-from flask import Flask
+from flask import Flask, request
 
 
 app = Flask(__name__)
 
-@app.route("/", methods =["GET"])
-def main():
-    return "<h1 style='Blue:red'>Hello World!</h1>"
+@app.route("/datasets", methods =["GET"])
+def datasets():
+    if request.headers['Accept'] == "application/ld+json":
+        return get_json(request)
+    else:
```

Comment:
you can just `return "", 406` in place of the else and lose the "not_recognised" function.

we need a "get_json" function ready for when things get complicated later, in the case of it not being a supported accept header we'll always just return a 406, no need to defer that to a function.